src/dsa.ts

```diff
@@ -133,7 +133,7 @@ export class DSA {
 
   public async getAcocuntIdDetails(instanceId: Instance['id']) {
     try {
-      const contract = new this.web3.eth.Contract(Abi.read.core, Addresses.read.core)
+      const contract = new this.web3.eth.Contract(Abi.read, Addresses.core.read)
```

Comment:
@DakshMiglani Let's use `Abi.read.core`?

Reply:
![image](https://user-images.githubusercontent.com/23355449/113511373-17579b00-957d-11eb-847b-3c79790dd720.png)

Abi.read.core doesn't exist with the new file structure from ba2dfa6

Reply:
Sorry! I meant `Abi.core.read`

Reply:
Yes, that's exactly what I used!